docs/07-deployments/04-deploying-to-aws.md

```diff
@@ -89,7 +89,21 @@ If you deploy your servers in a region other than Oregon (us-west-2), you will n
 
 By default, the Terraform scripts are configured to use a minimal setup to reduce costs for running your Serverpod. You can quickly turn on additional features, such as enabling Redis or adding a staging server by changing values in the script. You can also change these values later and redo the deployment step.
 
-Finally, to complete your Serverpod configuration, you will need to edit the `config/staging.yaml` and `config/production.yaml` files. In these files, you replace the `examplepod.com` domain with the domain you are using for your server.
+Finally, to complete your Serverpod configuration, you will need to edit the `config/staging.yaml` and `config/production.yaml` files. In these files you should:
+
+1. replace the `examplepod.com` domain with the domain you are using for your server.
+
+2. replace the database with the database url from the RDS. Replace the url with the corresponding environment yaml file in `mypod_server/config` under the `database.host` section
+
+```bash
+aws rds describe-db-instances --db-instance-identifier ${YOUR_DB_INSTANCE_ID} | jq ".DBInstances.[0].Endpoint.Address" -r
+```
```

Comment:
Suggestion: Add a description on what this does, I'm assuming it fetches the database URL, but spell that out in text!